templates/rhpam73-authoring-ha.yaml

```diff
@@ -260,21 +260,26 @@ parameters:
   value: ''
   example: "-Xms1024m -Xmx1024m"
   required: false
-- displayName: AMQ ImageStream Namespace
-  description: Namespace in which the ImageStream for the AMQ image is installed. Default is "openshift".
-  name: AMQ_IMAGE_STREAM_NAMESPACE
+- displayName: AMQ Scaledown Controller ImageStream Namespace
+  description: Namespace in which the ImageStream for the AMQ Scaledown Controller image is installed. Default is "openshift".
+  name: AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAMESPACE
   value: "openshift"
   required: true
-- displayName: AMQ ImageStream Name
-  description: The name of the image stream to use for the AMQ broker. Default is "amq-broker72-openshift".
-  name: AMQ_IMAGE_STREAM_NAME
-  value: "amq-broker72-openshift"
+- displayName: AMQ Scaledown Controller ImageStream Name
+  description: The name of the image stream to use for the AMQ Scaledown Controller. Default is "amq-scaledown-controller-openshift".
+  name: AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_NAME
+  value: "amq-scaledown-controller-openshift"
   required: true
-- displayName: AMQ ImageStream Tag
-  description: The AMQ image stream tag. Default is "1.0".
-  name: AMQ_IMAGE_STREAM_TAG
-  value: "1.1"
+- displayName: AMQ Scaledown Controller ImageStream Tag
+  description: The AMQ image stream tag. Default is "0.4".
+  name: AMQ_SCALEDOWN_CONTROLLER_IMAGE_STREAM_TAG
   required: true
+  value: "0.4"
+- description: Broker Image
+  displayName: Image
+  name: IMAGE
```

Comment:
A template parameter of name "IMAGE" is not distinct nor descriptive enough for a template that uses multiple images. Please rename to something else... perhaps "AMQ_BROKER_IMAGE"?